### PR TITLE
Interactive services SDK sample

### PR DESCRIPTION
**Pull request by shouryaj**
# Description

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    "        \"My_vscode\": JobService(\n",
+    "            job_service_type=\"vs_code\",\n",
+    "        ),\n",
+    "        \"My_tensorboard\": JobService(\n",
+    "            job_service_type=\"tensor_board\",\n",
+    "            properties={\n",
+    "                \"logDir\": \"output/tblogs\"  # relative path of Tensorboard logs (same as in your training script)\n",
+    "            },\n",
+    "        ),\n",
+    "        \"My_ssh\": JobService(\n",
```
Comment (srsaggam) [inline sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb:149]: Planning to comment this out?

File: sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    "        \"My_jupyterlab\": JobService(\n",
+    "            job_service_type=\"jupyter_lab\",\n",
+    "        ),\n",
+    "        \"My_vscode\": JobService(\n",
+    "            job_service_type=\"vs_code\",\n",
+    "        ),\n",
+    "        \"My_tensorboard\": JobService(\n",
+    "            job_service_type=\"tensor_board\",\n",
+    "            properties={\n",
+    "                \"logDir\": \"output/tblogs\"  # relative path of Tensorboard logs (same as in your training script)\n",
```
Comment (srsaggam) [inline sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb:146]: output-> outputs?

File: sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb (hunk 1 above)
Comment (shouryaj) [reply sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb:149]: the ssh piece?

Comment (srsaggam) [reply sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb:149]: yeah

**Review by harneetvirk — APPROVED** [review]

File: sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb [hunk 3]
```diff
@@ -0,0 +1,229 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Enable live debugging & monitoring for your training jobs\n",
```
Comment (balapv) [inline sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb:7]:
```suggestion
    "# Debug & monitor your training jobs\n",
```

File: sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+   "metadata": {},
+   "source": [
+    "# 2. Configure and run the Command\n",
+    "In this section we will configure and run a standalone job using the `command` class. The `command` class can be used to run standalone jobs and can also be used as a function inside pipelines.\n",
+    "\n",
+    "## 2.1 Configure the Command\n",
+    "The `command` allows user to configure the following key aspects.\n",
+    "- `code` - This is the path where the code to run the command is located\n",
+    "- `command` - This is the command that needs to be run. If you need to reserve your cluster for debugging or monitoring purposes, you can use the `sleep` command.\n",
+    "- `inputs` - This is the dictionary of inputs using name value pairs to the command. The key is a name for the input within the context of the job and the value is the input value. Inputs can be referenced in the `command` using the `${{inputs.<input_name>}}` expression. To use files or folders as inputs, we can use the `Input` class. The `Input` class supports three parameters:\n",
```
Comment (balapv) [inline sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb:106]: Can you please remove the inputs section? Its not used in the sample

File: sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+    "    - `mode` - \tMode of how the data should be delivered to the compute target. Allowed values are `ro_mount`, `rw_mount` and `download`. Default is `ro_mount`\n",
+    "- `environment` - This is the environment needed for the command to run. Curated or custom environments from the workspace can be used. Or a custom environment can be created and used as well. Check out the [environment](../../../../assets/environment/environment.ipynb) notebook for more examples.\n",
+    "- `compute` - The compute on which the command will run. In this example we are using a compute called `cpu-cluster` present in the workspace. You can replace it any other compute in the workspace. You can run it on the local machine by using `local` for the compute. This will run the command on the local machine and all the run details and output of the job will be uploaded to the Azure ML workspace.\n",
+    "- `distribution` - Distribution configuration for distributed training scenarios. Azure Machine Learning supports PyTorch, TensorFlow, and MPI-based distributed training. The allowed values are `PyTorch`, `TensorFlow` or `Mpi`.\n",
```
Comment (balapv) [inline sdk/python/jobs/single-step/debug-and-monitor/debug-and-monitor.ipynb:113]:
```suggestion
    "",
```

**Pull request merged**